web/book/tests/snapshots/snapshot__language-features__distinct__distinct-on__0.snap

```diff
@@ -0,0 +1,9 @@
+---
+source: web/book/tests/snapshot.rs
+expression: "prql target:sql.postgres\n\nfrom employees\ngroup department (\n  sort age\n  take 1\n)\n"
+---
+SELECT
+  DISTINCT ON (department) *
+FROM
+  employees
```

Comment:
@max-sixty I was looking back and noticed that this seems to have the `ORDER BY` clause removed compared to the one in the roadmap that was removed in this PR?

```sql
SELECT DISTINCT ON (department) *
FROM employees
ORDER BY department, age
```

Reply:
I have created a new issue #2711